libs/pages/services/src/lib/feature/page-job-create-feature/post-feature/post-feature.tsx

```diff
@@ -0,0 +1,265 @@
+import { APIVariableScopeEnum, JobRequest, VariableImportRequest } from 'qovery-typescript-axios'
+import { useEffect, useState } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { useNavigate, useParams } from 'react-router-dom'
+import { createApplication, postApplicationActionsDeploy } from '@qovery/domains/application'
+import { importEnvironmentVariables } from '@qovery/domains/environment-variable'
+import { selectAllRepository, selectOrganizationById } from '@qovery/domains/organization'
+import { ServiceTypeEnum } from '@qovery/shared/enums'
+import {
+  FlowVariableData,
+  JobConfigureData,
+  JobGeneralData,
+  JobResourcesData,
+  OrganizationEntity,
+  RepositoryEntity,
+} from '@qovery/shared/interfaces'
+import {
+  SERVICES_JOB_CREATION_CONFIGURE_URL,
+  SERVICES_JOB_CREATION_GENERAL_URL,
+  SERVICES_JOB_CREATION_RESOURCES_URL,
+  SERVICES_JOB_CREATION_VARIABLE_URL,
+  SERVICES_URL,
+} from '@qovery/shared/router'
+import { FunnelFlowBody } from '@qovery/shared/ui'
+import { buildGitRepoUrl, convertCpuToVCpu, useDocumentTitle } from '@qovery/shared/utils'
+import { AppDispatch, RootState } from '@qovery/store'
+import Post from '../../../ui/page-job-create/post/post'
```

Comment:
It confirms my request above having just that as a component name is quite weird I think !

Reply:
Will be changed as `StepSummary`